pkgs/applications/science/biology/cd-hit/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, zlib, perl }:
+
+stdenv.mkDerivation rec {
+  version = "4.8.1";
+  name = "cd-hit-${version}";
```

Comment:
You can just use `pname = "cd-hit"`, `mkDerivation` will add the version.

Reply:
Thanks!